src/Compiler/Microsoft.CodeAnalysis.Razor/src/ComponentDetectionConventions.cs

```diff
@@ -1,10 +1,7 @@
 ﻿// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
-#nullable enable
-
 using System;
```

Comment:
Because it was redundant. `Microsoft.CodeAnalysis.Razor` already has `<Nullable>enable</Nullable>` via `src\Compiler\Directory.Build.props`. The IDE actually grayed this out, so I removed it.